src/gocept/gtimelog/core.py

```diff
@@ -483,7 +487,7 @@ def load(self):
         groups = {}
         self.last_mtime = self.get_mtime()
         try:
-            for line in file(self.filename):
+            for line in open(self.filename, encoding='utf-8'):
```

Comment:
This should produce a deprecation warning because the file does not get closed.

Reply:
I re-factored most of the open statement into context manager.